src/utils.jsx

```diff
@@ -0,0 +1,8 @@
+export function obtenerItemsPorCategoria(lista, categoria, valor) {
+    const nuevaLista = lista.filter((item) => {
+      if (item[categoria] === valor) {
+        return item;
+      }
+    });
+    return nuevaLista;
+  }
```

Comment:
The function is great! but keep attention to indendentation, let's get prettier set up in your vs code to fix this in the future. No need to change this now.

Reply:
perfecto perfecto